src/SmartComponents/Inventory/applications/GeneralInformation.js

```diff
@@ -1,16 +1,117 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import { connect } from 'react-redux';
+import {
+    Card,
+    CardHeader,
+    CardBody,
+    TextContent,
+    Text,
+    TextVariants,
+    Grid,
+    GridItem,
+    GutterSize
+} from '@patternfly/react-core';
+import { Table } from '../../../PresentationalComponents/Table';
+import { Pagination } from '../../../PresentationalComponents/Pagination';
+import './general-information.scss';
 
-class GeneralInformation extends React.Component {
+class GeneralInformation extends Component {
     render() {
+        const { entity } = this.props;
+        const facts = entity.facts.qpc || entity.facts.inventory;
+        const timeZone = new Date(entity.created).toString().split(' GMT').slice(1)[0];
         return (
-            <div>General Information</div>
+            <Grid sm={ 6 } md={ 6 } lg={ 6 } gutter={ GutterSize.md }>
+                <GridItem>
+                    <TextContent>
+                        <Card>
+                            <CardHeader>
+                                <Text component={TextVariants.h1}>System</Text>
+                            </CardHeader>
+                            <CardBody>
+                                <div>
```

Comment:
Instead of using all these `<div>s` can we just use PF's `<Level>/<LevelItem>` component?